### PR TITLE
add support for --info to search command

### PR DESCRIPTION
**Pull request by luntain**

### CODE REVIEW

File: src/Action/Search.hs [hunk 1]
```diff
@@ -33,10 +34,22 @@ actionSearch Search{..} = replicateM_ repeat_ $ -- deliberately reopen the datab
...
-        when (hidden /= []) $ do
-            whenNormal $ putStrLn $ "-- plus more results not shown, pass --count=" ++ show (count+10) ++ " to see more"
+        if null res then
+            putStrLn "No results found"
+         else if info then do
+            let Target{..} = head res
+            putStrLn (unHTML targetItem)
+            let packageModule = map fst $ catMaybes [targetPackage, targetModule]
+            unless (null packageModule) $
+              putStrLn (unwords packageModule)
```
Comment (ndmitchell) [inline src/Action/Search.hs]: 4 spaces per indent as a preference

**Comment by ndmitchell** [comment]
Thanks! Looks perfect.

One very minor indentation issue, but sufficiently minor that I'm happy to leave it and if one day I'm working nearby I'll change it, and if not, I don't care super much. More a note for any future pull requests you submit :)

**Comment by luntain** [comment]
I was trying to maintain the fours spaces indent, must have forgotten and
reverted to my habits of 2 spaces. Should have changed the editor settings.

**Comment by luntain** [comment]
Just had a second look at the code, there's just on line indented with 2
spaces. The rest may look off because the else's are one space off because
the project compiles with Haskell98. Anyway, thanks for the tool!

**Comment by ndmitchell** [comment]
Yep, I was just referring to that one line, I appreciate the silly if/then indentation is required.

**Pull request merged**